### PR TITLE
fix(FEC-4063): KUXRN-4063 once multicast stream fails to start playback due to negative start position we help it with seeking to 0

### PR DESCRIPTION
**Pull request by giladna**
fix(KUXRN-4063) - once multicast stream fails to start playback due to negative start position we help it with seeking to 0

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java [hunk 1]
```diff
@@ -942,7 +947,9 @@ private PlayerEngine.EventListener initEventListener() {
                 PKEvent event;
                 switch (eventType) {
                     case PLAYING:
-                        updateProgress();
+                        if (!PKMediaFormat.udp.equals(sourceConfig.mediaSource.getMediaFormat())) {
```
Comment (tomasgarba) [inline playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java]: For better readability add function isMulticast or isUDP. 🐶

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java [hunk 2]
```diff
@@ -1232,7 +1232,7 @@ public void seekTo(long position) {
             isSeeking = true;
             sendDistinctEvent(PlayerEvent.Type.SEEKING);
             profiler.onSeekRequested(position);
-            if (player.getDuration() == TIME_UNSET) {
+            if (player.getDuration() == TIME_UNSET && !PKMediaFormat.udp.equals(sourceConfig.mediaSource.getMediaFormat())) {
```
Comment (tomasgarba) [inline playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java]: 🐶

File: playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java (hunk 1 above)
Comment (giladna) [reply playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java]: done

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java (hunk 2 above)
Comment (giladna) [reply playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java]: done

**Review by tomasgarba — APPROVED** [review]

**Pull request merged**